common/static_modifiers/mec_asari_static_modifiers.txt

```diff
@@ -8,7 +8,7 @@ mec_asari_mod_pop_ardatyakshi = {
 # Thessia Origin - Stagnation
 
 mec_asari_mod_origin_thessia_stagnation = {
-	pop_growth_speed = -0.25
```

Comment:
Let's just merge this in with the BtR modifiers and get rid of this file

Reply:
rgr